core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -73,11 +73,11 @@ public DefaultAuthenticationServer(String hostName, AlluxioConfiguration conf) {
     mHostName = hostName;
     mConfiguration = conf;
     mCleanupIntervalMs =
-            conf.getMs(PropertyKey.AUTHENTICATION_INACTIVE_CHANNEL_REAUTHENTICATE_PERIOD);
+        conf.getMs(PropertyKey.AUTHENTICATION_INACTIVE_CHANNEL_REAUTHENTICATE_PERIOD);
```

Comment:
Shall we bump up the default reauthentication period now that the closed channels are cleaned up automatically?

Reply:
That would prolong duration at which clients could remain authenticated. Are we fine with that? I'm mostly in favor to revoke auth after a period of inactivity.

Reply:
yeah I would think the period should be somewhat larger than the time to complete a normal user task. That should be no more than a couple days. Any concern for making it 3 days?

Reply:
That should be OK.